examples/websocket/src/main/kotlin/cnt/websocket/MessageEnDecoder.kt

```diff
@@ -0,0 +1,34 @@
+package cnt.websocket
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import java.io.Reader
+import java.io.Writer
+import javax.websocket.Decoder
+import javax.websocket.Encoder
+import javax.websocket.EndpointConfig
+
+
+class MessageEnDecoder
+    : Encoder.TextStream<Message>, Decoder.TextStream<Message> {
+
+    private val objectMapper = ObjectMapper().apply {
```

Comment:
`ObjectMapper` uses a fluent syntax for high level configuration, so the `apply` should not be necessary.

```
ObjectMapper()
    .registerModule(..)
    . setDefault...
```